spec/tools/server_settings_replicator/server_settings_replicator_spec.rb

```diff
@@ -10,8 +10,13 @@
   describe "#replicate" do
     it "targets only other servers" do
       miq_server.add_settings_for_resource(settings)
+      expected_output = <<-MESSAGE.strip_heredoc
+        Replicating from server id=#{miq_server.id}, path=k1/k2 to 1 servers
+        Settings: {:k1=>{:k2=>{:k3=>"v3"}}}
+        Done
+      MESSAGE
```

Comment:
Squiggly heredoc is Ruby 2.3+ (supported), so prefer `<<~` instead of `strip_heredoc`

Reply:
_NickLaMuro requests changes to this pull request review_

This should be a rubocop rule change, and not something that should be enforced by code reviewers.   If you are making suggestions like this, you should be putting this up for review in the guides repo for our standard rubocop rule set, and not here.